### ISSUE TITLE
gemfile version does not work because of pcregrep not found

### ISSUE THREAD

**Issue by Slike9** [reviewdog/action-rubocop#31]
`gemfile` version does not work because of "pcregrep: not found"
```
/home/runner/work/_actions/reviewdog/action-rubocop/master/script.sh: 24: /home/runner/work/_actions/reviewdog/action-rubocop/master/script.sh: pcregrep: not found
  Cannot get the rubocop's version from Gemfile.lock. The latest version will be installed.
```

Tested on v1 and master. Seems the problem started from v1.8, after migrating from Dockerfile.

GH-workflow config:
```yaml
name: Lint
on:
  pull_request:
jobs:
  ruby:
    name: Ruby
    runs-on: ubuntu-latest
    steps:
      - name: Check out code
        uses: actions/checkout@v2
      - uses: ruby/setup-ruby@v1
      - name: Run Rubocop
        uses: reviewdog/action-rubocop@master
        with:
          rubocop_version: gemfile
          rubocop_extensions: rubocop-rails:gemfile
          github_token: ${{ secrets.github_token }}
          reporter: github-pr-review
```

Problem screenshot:
![Screenshot from 2021-02-20 01-24-24](https://user-images.githubusercontent.com/300597/108563152-71c4d100-731a-11eb-90f9-f3bddaaf90f0.png)

Action: https://github.com/Slike9/rails-test-prj/runs/1938560595?check_suite_focus=true

Possible solutions:
* Just add a note to README that you need to install `pcregrep` to use `gemfile` version in your GH-workflow.
* The action can install `pcregrep` for us. There are problems with portability. We can use some portable way of installing the tool. Or we can stick with ubuntu for example.
* Use something else then `pcregrep`, something more ubiquitous. Suspect we can use sed, grep, a Ruby-script, smth else there.
* smth else ...

**Comment by Slike9** [comment]
Using Ruby one-liner:
```
RUBOCOP_GEMFILE_VERSION=$(ruby -ne 'print $& if /^\s{4}rubocop\s\(\K.*(?=\))/' Gemfile.lock)
```

**Comment by mgrachev** [comment]
@Slike9 Hey! Thank you for your help!

**Comment by mgrachev** [comment]
Fixed in [v1.8.1](https://github.com/reviewdog/action-rubocop/releases/tag/v1.8.1). Thanks ✌️
